### ISSUE TITLE
Plugin Crash when creating awx_organization

### ISSUE THREAD

**Issue by Timmu91** [denouche/terraform-provider-awx#73]
I get the following error when I try to create a new awx_organization:
```
awx_organization.acc-test: Creating...
╷
│ Error: Plugin did not respond
│
│   with awx_organization.acc-test,
│   on test.tf line 1, in resource "awx_organization" "acc-test":
│    1: resource "awx_organization" "acc-test" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ApplyResourceChange call. The plugin logs may contain more details.
╵

Stack trace from the terraform-provider-awx_v0.24.3 plugin:

panic: interface conversion: interface {} is nil, not int

goroutine 65 [running]:
github.com/denouche/terraform-provider-awx/awx.resourceOrganizationsCreate({0xe293a8, 0xc0005dd980}, 0x0?, {0xbb8460?, 0xc0005a0000?})
        github.com/denouche/terraform-provider-awx/awx/resource_organization.go:91 +0x6d1
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0xc000149a40, {0xe293e0, 0xc0005dcea0}, 0xd?, {0xbb8460, 0xc0005a0000})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.30.0/helper/schema/resource.go:778 +0x12e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0xc000149a40, {0xe293e0, 0xc0005dcea0}, 0xc000418750, 0xc0004ae880, {0xbb8460, 0xc0005a0000})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.30.0/helper/schema/resource.go:909 +0xa7e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0xc000369f20, {0xe293e0?, 0xc0005dcd80?}, 0xc00007df90)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.30.0/helper/schema/grpc_provider.go:1072 +0xe8d
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ApplyResourceChange(0xc00020afa0, {0xe293e0?, 0xc0005dc360?}, 0xc0001620e0)
        github.com/hashicorp/terraform-plugin-go@v0.19.0/tfprotov5/tf5server/server.go:859 +0x574
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0xcd29a0?, 0xc00020afa0}, {0xe293e0, 0xc0005dc360}, 0xc000162070, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.19.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:467 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0002061e0, {0xe2d0b8, 0xc00008a680}, 0xc00059ac60, 0xc000373cb0, 0x131ad78, 0x0)
        google.golang.org/grpc@v1.57.1/server.go:1358 +0xe23
google.golang.org/grpc.(*Server).handleStream(0xc0002061e0, {0xe2d0b8, 0xc00008a680}, 0xc00059ac60, 0x0)
        google.golang.org/grpc@v1.57.1/server.go:1735 +0xa36
google.golang.org/grpc.(*Server).serveStreams.func1.1()
        google.golang.org/grpc@v1.57.1/server.go:970 +0xca
created by google.golang.org/grpc.(*Server).serveStreams.func1
        google.golang.org/grpc@v1.57.1/server.go:981 +0x15c

Error: The terraform-provider-awx_v0.24.3 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```
I use a freshly installed awx instance. (AWX-Operator 2.15.0  ,AWX 24.2.0)
Maybe related to #66

**Comment by lavahot** [comment]
Can you drop in your config for the org?

**Comment by Timmu91** [comment]
Do you mean the tf file?
```
terraform {
  required_providers {
    awx = {
      source = "denouche/awx"
      version = "0.24.3"
    }
  }
}

provider "awx" {
  hostname = "################"
  token    = "################"
}

resource "awx_organization" "acc-test" {
  name            = "acc-test"
}

```

**Comment by denouche** [comment]
Fixed with #77
